### ISSUE TITLE
Stylus catched some errors with the CSS

### ISSUE THREAD

**Issue by MrLabRat** [StylishThemes/GitHub-Dark#534]
warning | 925 | : | 5 | Expected (static \| relative \| absolute \| fixed) but found 'sticky'.
warning | 926 | : | 5 | Expected (static \| relative \| absolute \| fixed) but found '-webkit-sticky'.
warning | 931 | : | 5 | Expected (static \| relative \| absolute \| fixed) but found 'sticky'.
warning | 932 | : | 5 | Expected (static \| relative \| absolute \| fixed) but found '-webkit-sticky'.
warning | 965 | : | 5 | Duplicate property 'background-color' found.
warning | 1239 | : | 14 | Expected (static \| relative \| absolute \| fixed) but found 'sticky'.
warning | 1241 | : | 7 | Expected (static \| relative \| absolute \| fixed) but found 'sticky'.
warning | 3283 | : | 5 | Duplicate property 'background' found.

*************************************************************************************************

I'm using Stylus 1.1.3.1 with Firefox 55.0.3 (64-bit) and latest GitHub-Dark theme

Thanks!

**Comment by Mottie** [comment]
Hi @MrLabRat!

Those are CSSLint issues. The CSSLint core isn't getting updated, but the owners of Stylus (and me) have addressed this and added support for `position: sticky`.

The two duplicate properties are due to the way we set up GitHub-Dark... within our style you'll find two entries that use this similar format:

```css
.selector {
  background-color: rgba(65, 131, 196, .2) !important;
  background-color: rgba(/*[[base-color-rgb]]*/, .2) !important;
}
```

What happens is that Userstyles replaces the `/*[[base-color-rgb]]*/` with the custom color you select when installing the style, but if you keep the default color, you'll end up with a duplicate `rgba()` color. This is why CSSLint is reporting duplicate properties.

So, bottom line, please ignore these errors. In the next version of Stylus, you'll still see the duplicate warning, but you'll be able to configure the messages to disable that warning.

**Comment by silverwind** [comment]
Maybe the Stylus default linter config should be adjusted to ignore duplicates, seeing that it is probably pretty common among userstyles.

**Comment by Mottie** [comment]
Like I said, you can change the CSSLint configuration in the next update... I also added Stylelint support, and the ability to configure it as well - see https://github.com/openstyles/stylus/pull/150.

**Comment by MrLabRat** [comment]
Thanks for all your replies! :D

**Comment by Mottie** [comment]
[Stylus v1.1.4.1 for Firefox](https://addons.mozilla.org/en-US/firefox/addon/styl-us/) is now available.

**Comment by MrLabRat** [comment]
Yeah, I've already have it on auto-update 👍 

Thanks!
